addons/escoria-core/game/core-scripts/esc/commands/camera_push.gd

```diff
@@ -110,3 +98,29 @@ func interrupt():
 		self,
 		"[%s] interrupt() function not implemented." % get_command_name()
 	)
+
+
+func _generate_viewport_warning(new_pos: Vector2, camera: ESCCamera) -> void:
```

Comment:
This method is identical in other camera-related commands. Maybe it would be interesting to define a new `ESCCameraBaseCommand` class extending `ESCBaseCommand`, and make all camera commands extend this new class.